libs/community/langchain_community/document_loaders/apache_opendal_s3.py

```diff
@@ -0,0 +1,103 @@
+from __future__ import annotations
+
+import os
+import tempfile
+from typing import Any, Callable, List, Optional
+
+from langchain_community.document_loaders.unstructured import UnstructuredBaseLoader
+
+
+class ApacheOpendalS3FileLoader(UnstructuredBaseLoader):
```

Comment:
should we include Unstructured in the name to clarify that this is using unstructured under the hood?

Reply:
> should we include Unstructured in the name to clarify that this is using unstructured under the hood?

In fact, I see a lot of document_loader using `UnstructuredBaseLoader`, like `S3FileLoader`,`GCSFileLoader` etc. 
Do they all need to be modified?

Reply:
hm fair enough

Reply:
> hm fair enough

Do I need to modify them in this PR? Or should I open another PR to fix them?

Reply:
> should we include Unstructured in the name to clarify that this is using unstructured under the hood?

Done. PTAL